tests/unit/assert_test.sh

```diff
@@ -81,91 +81,91 @@ function test_unsuccessful_assertNotMatches() {
 }
 
 function test_successful_assertExitCode() {
-  function fake_function() {
+  function fake_function1() {
```

Comment:
why did we change the name to the functions on each test?

Reply:
This might be due to a planning oversight. We want to find the duplicated test functions, not all functions. I believe adding to the regular expression that filters the functions to only search for those starting with test_ would be sufficient.

Reply:
Great idea 💡  I will do that